.github/workflows/sdist.yml

```diff
@@ -0,0 +1,56 @@
+name: sdist
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - 1.2.x
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: sdist test
+    timeout-minutes: 60
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install numpy cython python-dateutil pytz hypothesis pytest pytest-xdist
+        pip list
+
+    - name: Build Pandas sdist
+      run: |
+        python setup.py build_ext -q -j2
```

Comment:
I don't think we need to build pandas before creating the sdist? (since we no longer include the cythonized files)